### PR TITLE
Create ui-c8y-1018-503-129-fixed-issue-with-info-gauge-resizing-7491.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61448](https://cumulocity.atlassian.net/browse/MTM-61448)
Original PR: [7491](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7491)

[MTM-61448]: https://cumulocity.atlassian.net/browse/MTM-61448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-129-fixed-issue-with-info-gauge-resizing-7491.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed issue with info gauge resizing. (#7491)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-129-fixed-issue-with-info-gauge-resizing-7491.md]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed info gauge resizing issue
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-129-fixed-issue-with-info-gauge-resizing-7491.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61448
+version: 1018.503.129
+---
+Fixed issue with info gauge resizing. (#7491)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-129-fixed-issue-with-info-gauge-resizing-7491.md]: The Claude AI suggested to use the following body:
```suggestion
Info gauges in dashboards were not resizing correctly when changing the browser window size, leading to a poor user experience. This issue has been fixed and info gauges now properly resize to fit the available space when the browser window is resized. This change improves the usability and appearance of dashboards containing info gauges for all users.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@eniosultan Please check this PR

File: content/change-logs/application-enablement/ui-c8y-1018-503-129-fixed-issue-with-info-gauge-resizing-7491.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed info gauge resizing issue
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-129-fixed-issue-with-info-gauge-resizing-7491.md]:
```suggestion
title: Info gauges resize properly when the window size changes
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**